<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Lockins</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Reference (start it first)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="LockinGui" name="lockin_ref" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Reflexion (start it second)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="LockinGui" name="lockin_sig" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Fit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="XYView" name="graphicsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_start_fit">
            <property name="text">
             <string>(re)Start fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop_fit">
            <property name="text">
             <string>Stop fit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>719</width>
             <height>390</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fit data from time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_from">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>to time</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_to">
                <property name="text">
                 <string>1e15</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="2">
               <widget class="QLineEdit" name="lineEdit_std_layer_index">
                <property name="text">
                 <string>0.1</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLineEdit" name="lineEdit_std_deposition_rate">
                <property name="text">
                 <string>0.0001</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_5">
                <property name="statusTip">
                 <string>0 mean S polarization and 1 mean P polarization</string>
                </property>
                <property name="text">
                 <string>Polarization</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="statusTip">
                 <string>0 mean normal and 90 mean grazing</string>
                </property>
                <property name="text">
                 <string>Angle of incidence</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_layer_index">
                <property name="text">
                 <string>1.4</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_angle">
                <property name="text">
                 <string>40</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QLineEdit" name="lineEdit_std_polarization">
                <property name="text">
                 <string>0.5</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_7">
                <property name="statusTip">
                 <string>Time in seconds during which the deposition already started</string>
                </property>
                <property name="text">
                 <string>Time offset</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="lineEdit_deposition_rate">
                <property name="text">
                 <string>0.0001</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Substrate index</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Layer index</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="lineEdit_std_substrate_index">
                <property name="text">
                 <string>0.1</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="label_layer_index">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="label_polarization">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QLineEdit" name="lineEdit_std_time_offset">
                <property name="text">
                 <string>1000</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Substrate absorb.</string>
                </property>
               </widget>
              </item>
              <item row="7" column="3">
               <widget class="QLabel" name="label_time_offset">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="lineEdit_std_angle">
                <property name="text">
                 <string>5</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLineEdit" name="lineEdit_time_offset">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="label_angle">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_polarization">
                <property name="text">
                 <string>0.5</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_substrate_index">
                <property name="minimumSize">
                 <size>
                  <width>110</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_6">
                <property name="statusTip">
                 <string>Rate in wavelength per seconds</string>
                </property>
                <property name="text">
                 <string>Deposition rate</string>
                </property>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QLabel" name="label_deposition_rate">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_substrate_index">
                <property name="text">
                 <string>1.2</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_8">
                <property name="statusTip">
                 <string>[global factor] * [reflectance] + [global offset]</string>
                </property>
                <property name="text">
                 <string>Global factor</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="lineEdit_global_factor">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item row="8" column="2">
               <widget class="QLineEdit" name="lineEdit_std_global_factor">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item row="8" column="3">
               <widget class="QLabel" name="label_global_factor">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_substrate_abs">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_substrate_abs">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="lineEdit_std_substrate_abs">
                <property name="text">
                 <string>0.01</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Layer absorb.</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_layer_abs">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLineEdit" name="lineEdit_std_layer_abs">
                <property name="text">
                 <string>0.01</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_layer_abs">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_11">
                <property name="statusTip">
                 <string>[global factor] * [reflectance] + [global offset]</string>
                </property>
                <property name="text">
                 <string>Global offset</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QLineEdit" name="lineEdit_global_offset">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="9" column="2">
               <widget class="QLineEdit" name="lineEdit_std_global_offset">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="9" column="3">
               <widget class="QLabel" name="label_global_offset">
                <property name="text">
                 <string>&lt;no value&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_ratio"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_ratio">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LockinGui</class>
   <extends>QWidget</extends>
   <header>lockin2/src/lockin_gui.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XYView</class>
   <extends>QGraphicsView</extends>
   <header>lockin2/xygraph/xyview.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
